src/crc.ts

```diff
@@ -0,0 +1,11 @@
+import { crc32 } from 'crc';
+
+class CRC {
+    public static crc32(data: string): number {
+        const buffer = Buffer.from(data, 'utf16le');
```

Comment:
Can this cause any issues if the string encoding isn't `utf16le`? According to the [crc documentation](https://www.npmjs.com/package/crc#usage), they also support strings directly. So is there even a need to convert it to a buffer first?

Reply:
This is how the NI checksum is generated. It determines the checksum from utf16le string. Don't know why it was implemented that way in the past.

Reply:
Maybe it is worth to add a comment for this.